etcd-manager/.gitignore

```diff
@@ -20,4 +20,4 @@ bazel-testlogs
 dist/
 
 # Don't ignore vendoring
-!vendor/**
+!vendor/
```

Comment:
Why is this changing?

Reply:
lgtm - I would have thought they would be the same based on the docs, but I can confirm this does (correctly) include the version directory.

I _think_ it’s because we need to match the parent .gitignore (or maybe the issue is the vendor directory _itself_ is not toggled by `vendor/**`, but anyway this now lgtm.)